docs/introduction/developer-quickstart.md

```diff
@@ -6,22 +6,24 @@ sidebar_position: 4
 
 Obscuro executes bytecode in the EVM no different than Ethereum (or Arbitrum), so as a developer you can use all your existing code and tooling.
```

Comment:
I would change this line to:

Obscuro performs bytecode execution in the EVM identically to Ethereum, allowing developers to leverage their existing codebase and tools.

Reply:
Fixed